pkg/manifest/local_server.go

```diff
@@ -24,16 +31,171 @@ type LocalConfigStore struct {
 	Contents map[string]string `toml:"contents,omitempty"`
 }
 
-// LocalKVStore represents an kv_store to be mocked by the local testing server.
-type LocalKVStore struct {
+// KVStoreArrayEntry represents an array-based key/value store entries.
+// It expects a key plus either a data or file field.
+type KVStoreArrayEntry struct {
```

Comment:
Does this need to be extended to support 'Metadata', as was done in https://github.com/fastly/Viceroy/pull/459?

Reply:
Yes, I've pushed a commit to support it, see my newest push.